src/components/TestRail/ApiClient.js

```diff
@@ -15,6 +16,38 @@ class ApiClient {
         this.baseUrl = `https://${domain}/index.php?/api/v2`;
     }
 
+/**
+ *
+ * @param slug
+ * @returns {Promise<AxiosResponse<any>>}
+ */
+getData(slug, opts={}) {
+    const fullUrl = this.baseUrl + slug;
+    return axios({
+        method: 'get',
+        url: fullUrl,
+        headers: {
+            'Content-Type': 'application/json',
+        },
+        auth: {
+            username: this.username,
+            password: this.password,
+        },
+        params: { ...opts },
```

Comment:
```suggestion
        params: opts,
```
Or you can rename the `opts` argument to params so you can 
```suggestion
        params,
```